### PR TITLE
Andrew goenner

### PR DESCRIPTION
**Pull request by drewgoenner**
Completed all objectives and both stretches (objects.js and arrays.js).

### CODE REVIEW

**Comment by drewgoenner** [comment]
Sorry, the I did the VSCode stretches for objects and function-conversion. There was no array stretch in VSCode.

File: assignments/arrays.js [hunk 1]
```diff
@@ -63,32 +63,51 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
...
 // ==== Challenge 6 ====
 // A buyer is interested in seeing only BMW and Audi cars within the inventory.  Return an array that only contains BMW and Audi cars.  Once you have populated the BMWAndAudi array, use JSON.stringify() to show the results of the array in the console.
 let BMWAndAudi = [];
-console.log();
+for (i=0; i< inventory.length; i++) {
+    if (inventory[i].car_make === "BMW" || inventory[i].car_make === "Audi") {
+        BMWAndAudi.push(inventory[i]);
+    }
+}
+console.log(JSON.stringify(BMWAndAudi));
```
Comment (thereactgirl) [inline assignments/arrays.js:113]: The arrays.js file looks top notch! I would not have done it any differently!

File: assignments/function-conversion.js [hunk 2]
```diff
@@ -27,4 +43,8 @@
 // const triple = exampleArray.map(function (num) {
 //   return num * 3;
 // });
-// console.log(triple);
\ No newline at end of file
+// console.log(triple);
+
+exampleArray = [1,2,3,4];
+const triple = exampleArray.map((num) => num * 3);
+console.log(triple);
```
Comment (thereactgirl) [inline assignments/function-conversion.js:50]: Awesome job on the function conversions!

**Review by thereactgirl — COMMENTED** [review]
I did not find a single error Andrew congrats! 3 stars!

**Pull request merged**